homeassistant/components/v2c/strings.json

```diff
@@ -52,6 +52,18 @@
     "switch": {
       "paused": {
         "name": "Pause session"
+      },
+      "locked": {
+        "name": "Lock EVSE"
+      },
+      "timer": {
+        "name": "Charge Point Timer"
+      },
```

Comment:
Mind the casing

Reply:
Was following API documentation file 😓  -> Fixed